src/qtiCommonRenderer/renderers/Figure.js

```diff
@@ -40,10 +48,10 @@ export default {
         elem.attributes.showFigure = showFigure;
         return data;
     },
-    render: function(figure) {
+    render(figure) {
         const $figure = containerHelper.get(figure);
         const $img = $figure.find('img');
-        if ($img.length && $figure.prop('tagName') === 'FIGURE') {
+        if ($img.length && (DISABLE_FIGURE_WIDGET || $figure.prop('tagName') === 'FIGURE')) {
```

Comment:
Should not be a negation here as by default this feature is enabled?

```suggestion
        if ($img.length && !DISABLE_FIGURE_WIDGET && $figure.prop('tagName') === 'FIGURE') {
```

Reply:
No, because, if the feature is disabled we want the code to behave as before. We need to compare with the code before the feature was added, not with the last version.

See: https://github.com/oat-sa/tao-item-runner-qti-fe/pull/374/files#diff-c7cfdcd90c13f0cc2707f074e1f794aa1c2b7e66d160b361156c0cc2beb1b5a7R29